### ISSUE TITLE
The output of `rvm list known` needs to be updated

### ISSUE THREAD

**Issue by davidkrider** [rvm/rvm#5358]
I've put off upgrading to Ruby > 2.7 because 3.0 was having trouble installing, and I didn't know other versions were available. I only noticed that they were when I was following a link to download a specific version of 2.7 that I found in another thread (about the SSL problem). When someone runs `rvm list known`, they get this, on either stable or latest:

```
# MRI Rubies
[ruby-]1.8.6[-p420]
[ruby-]1.8.7[-head] # security released on head
...
[ruby-]2.6[.6]
[ruby-]2.7[.2]
[ruby-]3[.0.0]
ruby-head
```

There's no mention that 3.1 and 3.2 are available! I upgraded 5 machines across "all" 3 OS's to 3.2.2 yesterday. Thank you!

**Comment by Meekohi** [comment]
Agreed, this also makes `rvm install ruby --latest` install the wrong version of ruby.

**Comment by hmdne** [comment]
Looks to me like the `stable` version of RVM was last released in Jan 2021

**Comment by elichad** [comment]
Bumping this - as a new RVM/Ruby user wanting to upgrade from 3.0 to >3.1 I found this lack of information/support on `stable` extremely confusing. Can a new `stable` release be made?
